### DISCUSSION TITLE
Welcome to covid-rt-estimates Discussions!

### DISCUSSION THREAD

**Discussion by seabbs** [epiforecasts/covid-rt-estimates#163] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mzhangzhicheng** [comment]
Hello Sam, I am lucas, a university student in China. I would like to use this data for a survey. Could you please add the subnational data of China?
